spec/spec-compat.md

```diff
@@ -0,0 +1,139 @@
+
+# Wasm Image Specification v0.0.0
+
+## Introduction:
+
+This document describes a varient of [Wasm Artifact Image Specification](spec.md), which leverages the compatible layer media types. Here, we omit definition and terminology explained in [Wasm Artifact Image Specification](spec.md). 
+
+We call this variant "compat", and the spec in [Wasm Artifact Image Specification](spec.md) "oci".
+
+## Description
+
+This *compat* variant makes use of compatible media type for layers, and is not based on custom OCI Artifact media types. This way users can oeperate with standard tools such as docker, podman, buildah, etc.
```

Comment:
typo: `operate`

maybe also mention that this is strictly for compatibility, ie for registries that don't yet support custom mediaTypes in OCI images, or for existing toolchains that require standard docker images

Reply:
done! https://github.com/solo-io/wasm/pull/265/commits/d06e5880695665da98d90f770a9c741ba94ba2fa